spec/lib/resources_spec.rb

```diff
@@ -649,21 +649,59 @@
     end
 
     context 'when parameters are passed in' do
-      subject(:beta_tag_parameters) { Cerner::Resources::Helpers.beta_tag(action: action) }
+      subject(:beta_tag_parameters) { Cerner::Resources::Helpers.beta_tag(action: action, known_issues: issues) }
+      let(:issues) { %w[hi hello] }
+      let(:beta_tag_div) do
+        '<div class="beta-tag"></div>This Resource is still under development.<p>Known Issues:</p><ul><li>hi</li>'\
+        '<li>hello</li></ul></p>'
+      end
 
       context 'when action is false' do
         let(:action) { false }
 
-        it 'returns the beta_tag div for a Resource' do
-          expect(beta_tag_parameters).to eq(beta_tag_div)
+        context 'when known issues are not present' do
+          let(:issues) { nil }
+          let(:beta_tag_div) do
+            '<div class="beta-tag"><p>This Resource is still under development.</p></div>'
+          end
+
+          it 'returns the beta_tag div for a Resource' do
+            expect(beta_tag_parameters).to eq(beta_tag_div)
+          end
+        end
+
+        context 'when known issues are present' do
+          let(:beta_tag_div) do
+            '<div class="beta-tag"><p>This Resource is still under development.</p><p>Known Issues:</p><ul>'\
+            '<li>hi</li><li>hello</li></ul></div>'
+          end
+
+          it 'returns the beta_tag div for a Resource' do
```

Comment:
let's add 'with known issues' at the end of context here and on line 702

Reply:
[updated](https://github.com/cerner/fhir.cerner.com/pull/608/commits/726786db1ff6e7fe0226f1dbe2b2f6a9bb0a17be#diff-3a0461fbcd80bc81722ffd0c7101a7df86a5124d8d677ecc79fba7ccdccacad1R679)